lib/composer.js

```diff
@@ -14,11 +15,16 @@ import MagicParser from './parsers/magic-parser'
 export default class Composer extends Disposable {
   disposables = new CompositeDisposable()
   cachedBuildStates = new Map()
+  cachedDicy = new Map()
 
   constructor () {
     super(() => this.disposables.dispose())
     this.disposables.add(atom.config.onDidChange('latex', () => {
       this.rebuildCompleted = new Set()
+      const dicy = this.cachedDicy.values().next().value
+      if (dicy) {
+        dicy.updateOptions(this.getDicyOptions(), true)
+      }
```

Comment:
OK, that is not at all clear from the code, so I think this logic needs to be extracted into a function with a descriptive name. Or at least this needs a comment documenting that behavior.